wcc-contentful-app/lib/wcc/contentful/model/section_contact_form.rb

```diff
@@ -2,7 +2,7 @@
 
 class WCC::Contentful::Model::SectionContactForm < WCC::Contentful::Model
   def send_email(data)
-    ::WCC::Contentful::App::ContactMailer.contact_form_email(to_email(data), data).deliver
+    ::WCC::Contentful::App::ContactMailer.contact_form_email(data[:recipient_email], data).deliver
```

Comment:
One thing I think we need to consider - this is opening up a vector by which a malicious user (let's call her "eve") can override the email address to send to whoever she wants.  And that comes from our app.  So, if she's on the person page for "Alice", she simply goes into the HTML dom and changes the `recipient_email` parameter from "alice@watermark.org" to "bob@wherever.com".  She could also create a script to spam the form submit, it's just a post.  Now she's created a method to spam someone else's email from our Sendmail account.

We need to discuss how we're going to handle this scenario.